app/models/data/high_fail_questions.rb

```diff
@@ -27,12 +27,20 @@ def dashboard
 
       # @return [Hash{Array<String, String> => Integer}]
```

Comment:
Could you correct these docs that have been overlooked whilst you're in the methods please. The convention is to describe the types the collection can contain, not each type by element position.

`# @return [Hash{Array<String> => Integer}]`